soccer/SimFieldView.cpp

```diff
@@ -128,12 +129,12 @@ void SimFieldView::drawTeamSpace(QPainter& p) {
     // Simulator drag-to-shoot
     std::shared_ptr<LogFrame> frame = currentFrame();
     if (_dragMode == DRAG_SHOOT && frame) {
-        p.setPen(QPen(Qt::white, 0.1f));
+        p.setPen(QPen(Qt::white, 0.025f)); // TODO: smaller pen?
```

Comment:
You can go ahead and remove the TODO here, I think the pen size is fine how you have it.